draft-law-moq-warpstreamingformat.md

```diff
@@ -79,13 +80,40 @@ This document describes version 1 of the streaming format.
 
 This document uses the conventions detailed in Section 1.3 of {{!RFC9000}} when describing the binary encoding.
 
-# Packaging
+# Media packaging {#mediapackaging}
+WARP delivers CMAF-packaged media bitstreams. This specification references {{CMAFpackaging}} to define how CMAF packaged bitstreams are mapped to {{MoQTransport}} groups and objects.
 
-Each codec bitstream MUST be packaged in to a sequence of Objects within a separate track.
+Both CMAF Object mappings {{CMAFpackaging}} Section 4 are supported and a content producer may use either. To identify to consumers which object mapping mode is being used for a given Track, a new track field is defined as per table 1.
 
-Media tracks SHOULD be media-time aligned. CMAF {{CMAF}} Aligned Switching Sets meet this requirement. A receiver SHOULD be able to cleanly switch between media tracks at group boundaries.
+Table 1
+| Field                   |  Name                  | Required |  Location |  JSON type |      Definition            |
+|:========================|:=======================|:=========|:==========|:===========|:===========================|
+| WARP packaging mode     | warp-packaging         |  yes     |   RT      |  String    | See {{packagingmode}}      |
 
-Each group MUST be independently decodeable. Assigning a new group ID to each CMAF Fragment (see {{CMAF}} Sect 6.6.1) meets this requirement.
+
+## Packaging mode {#packagingmode}
+The packaging mode value is defined by Table 2.
+
+Table 2
+| warp-packing field value  |  Condition                         |                                              Explanation                                            |
+|:==========================|:===================================|:====================================================================================================|
+| frag-per-group            | {{CMAFpackaging}} 4.1 is active    |  Each CMAF Fragment is placed in a single MOQT Object and there is one MOQT Object per MOQT Group   |
+| chunk-per-object          | {{CMAFpackaging}} 4.2 is active    |  Each CMAF chunk is placed in a MOQT Object and there is one MOQT Group per CMAF Fragment           |
```

Comment:
I feel the wording is confusing here.

Reply:
I'm not sure what is confusing about "Each CMAF Fragment is placed in a single MOQT Object and there is one MOQT Object per MOQT Group"? CMAF Fragment, MOQAT Objects and MOQT Groups are all defined entities. Can you propose some language you consider less confusing?

Reply:
Received comment from Suhas via Webex "the above PR looks good Will .. i re-read once more and you are right". Will merge.